promotions/lib/solidus_promotions/promotion_map.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
```

Comment:
We could add some lines of documentation here to explain that this file is separate so that people can add their mapping in case they have custom rules/conditions or actions/benefits.

Reply:
Done